docs/Data loading, Migration/Import CSV data.md

```diff
@@ -1,104 +1,87 @@
-# Import CSV data
+# Import CSV Data
 
-MariaDB SkySQL customers can import data into a SkySQL service using the `LOAD DATA LOCAL INFILE` SQL statement:
+MariaDB SkySQL customers can import data into a SkySQL service using the `LOAD DATA LOCAL INFILE` SQL statement:
 
-- The `LOAD DATA LOCAL INFILE` statement can import data from TSV and CSV files
-- The `LOAD DATA LOCAL INFILE` statement can be executed by any client or connector
+- The `LOAD DATA LOCAL INFILE` statement can import data from TSV and CSV files.
+- The `LOAD DATA LOCAL INFILE` statement can be executed by any client or connector.
 
 !!! Note
-    Make sure your schema is already created in the database. If you need to import entire databases or create tables, you should use mariab-import.
+    Ensure your schema is already created in the database. For importing entire databases or creating tables, consider using `mariadb-import`.
 
+## Enable Local Infiles
 
-## **Enable Local Infiles**
+Support for local infiles must be enabled on both the client side and the SkySQL service.
 
-Support for local infiles must be enabled on the client side and on the SkySQL service.
+### Enable Local Infiles on the Client or Connector
 
-### **Enable Local Infiles on the Client or Connector**
+To execute the [`LOAD DATA LOCAL INFILE`](https://mariadb.com/docs/skysql-dbaas/ref/mdb/sql-statements/LOAD_DATA_INFILE/) statement, most clients and connectors require a specific option to be enabled.
 
-To execute the [`LOAD DATA LOCAL INFILE`](https://mariadb.com/docs/skysql-dbaas/ref/mdb/sql-statements/LOAD_DATA_INFILE/) statement, most clients and connectors require a specific option to be enabled.
+If you are using `mariadb` client, the [`--local-infile` option](https://mariadb.com/docs/skysql-dbaas/ref/mdb/cli/mariadb/local-infile/) must be specified.
 
-If you are using `mariadb` client, the [`--local-infile` option](https://mariadb.com/docs/skysql-dbaas/ref/mdb/cli/mariadb/local-infile/) must be specified.
-
-### **Enable Local Infiles in SkySQL**
+### Enable Local Infiles in SkySQL
 
 Support for local infiles must be enabled on the SkySQL service.
 
-For SkySQL services that use MariaDB Enterprise Server and MariaDB Enterprise ColumnStore, the [local_infile system variable](https://mariadb.com/docs/skysql-dbaas/ref/mdb/system-variables/local_infile/) must be enabled:
+For SkySQL services that use MariaDB Enterprise Server and MariaDB Enterprise ColumnStore, the [`local_infile` system variable](https://mariadb.com/docs/skysql-dbaas/ref/mdb/system-variables/local_infile/) must be enabled:
```

Comment:
As mentioned please remove all links to the old SkySQL. The links can stop working anytime and will also confuse the user.